src/AdminCommands.ts

```diff
@@ -42,8 +42,8 @@ export class AdminCommands {
     ];
     constructor(private main: Main) {
         this.yargs = yargs.parserConfiguration({})
-        .version(false)
-        .help(false); // We provide our own help, and version is not required.
+                                                .version(false)
+                                                .help(false); // We provide our own help, and version is not required.
```

Comment:
wtf happened to the indentation!?

4 spaces please. Not zero, and not a million.

Reply:
You should probably be more explicit :p